deb-x64/Dockerfile

```
@@ -104,6 +104,10 @@ RUN curl -Sl -O https://dd-agent-omnibus.s3.amazonaws.com/kernel-4.9-headers-deb
 # Download and install golangci-lint
 RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $GOPATH/bin v1.21.0
 
+# Rust is needed to compile some python libs
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain 1.41.0
```

```suggestion
RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain ${RUST_VERSION}
```

Also, why not use 1.53 (and bump the version on builders too)?

---

Because this was an old branch I already had :) I've bumped it to 1.53 like you suggested but this might not be the solution we go for since rust can't be installed in suse11.